lib/linguist/languages.yml

```diff
@@ -3858,6 +3858,15 @@ Literate Haskell:
   codemirror_mode: haskell-literate
   codemirror_mime_type: text/x-literate-haskell
   language_id: 207
+LiveCode Script:
+  type: programming
+  color: "#0c5ba5"
+  extensions:
+  - ".livecodescript"
+  - ".lc"
```

Comment:
The non-LiveCode script usage of this extension far outweighs the LiveCode Script usage. Linguist doesn't know of another language with this extension so all those files will be incorrectly identified.

You have two choices:

1. Identify at least one other language that uses that extension, that meets our usage requirements, and add it as part of this PR, along with a heuristic to distinguish the two, and removing the "hello world" example we [expressly state](https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language) will be rejected, or
2. Remove this extension and samples and users can then use an override until such time as someone does the first option.